### DISCUSSION TITLE
¡Sistema de clanes!

### DISCUSSION THREAD

**Discussion by Darkusito** [zebnat/morpho-dimension#19] [Ideas]
En la época dorada de **Macrojuegos** muchos usuarios creaban clanes con el fin de **rolear batallas con otros usuarios**, recuerdo haber participado en un par de estos y era bastante genial. Por lo cual se me ocurrió la idea de **crear un sistema de clanes oficial dentro de MorphoD** ¿Cómo funcionaría esto?

Pues, para empezar habría que comprar **la casa del clan**. (Sería como un perfil mas pero con diferentes características, como por ejemplo que varios usuarios puedan modificar su home si tienen los permisos) he incluso podrían crear logos de clan utilizando de base al águila, ya que es un avatar poco usado y en su momento se le usaba para ser imagen de los clanes.

Ya cuando los usuarios **tengan un clan propio o se hayan unido a alguno**, que entre estos **comiencen a competir entre si en un ranking algo parecido al original que conocemos,** pero en vez de estar basado en usuarios. Que estuviese basado en clanes.
de esta forma, al jugar aparte de conseguir monedas y experiencia, también le aportases al clan con **Puntos de clan** _(o algo parecido)_

De esta forma el clan **puede ir escalando niveles e ir recogiendo recompensas extras al ir subiendo de nivel**, pero que este se resetee cada cierto tiempo como si fuesen temporadas.

En mi mente suena bastante buena idea y fomentaría un poco mas el socializar con los demás usuarios. ¿Qué opinan ustedes?

**Comment by zebnat** [comment]
En lo que respecta al tema técnico, tiene bastante trabajo. Por ahora solamente yo estoy colaborando ya que se necesita un cierto nivel de Frontend Developer para aportar código y el proyecto ni siquiera está en fase beta aún, pero no hay ninguna limitación que impida que se pudiese hacer algo como esto.

Es una característica que pese a estar en el cliente/aplicación de morphoD, requeriría de unas cuantas adiciones en la base de datos del servidor de Panaworld, por lo que necesitaría confirmación de que no haya problema en generar esos datos.

De aprobarse, se pondría al final de la cola de características, ya que otras se pueden programar con mayor celeridad y ésta llevaría mucho más tiempo. Es decir, si se nos ocurren nuevas características más rápidas seguramente agarren prioridad.

En lo que respecta a detalles, animaría a la comunidad a aportar y detallar más, ya que se puede hacer de muchas formas. Por ejemplo se podría poner un "impuesto de clan", donde el clan decidiría qué % del oro generado deriva a la hucha del clan. También habría que analizar qué podría hacer el clan con ese dinero, ya que posibilidades son muchas, de modo que a la hora de programarse este sistema tendría como diferentes versiones que irían de menos a más. Es decir es un "entorno" al que se le pueden agregar muchas cosas o menos cosas. En el fondo un server de Discord podría ser considerado un clan, por lo tanto el sistema necesitará gestión de usuarios, roles, comunicación de clan, y más cosas.

Tendríamos que valorar si merece la pena dedicar tanto tiempo analizando la diversión que aportaría algo como esto. Morpho se creó para esto, pero aun así siempre daremos prioridad a características que aporten mucho o solucionen problemas graves y al mismo tiempo, no cuesten tanto de programar.

Esperamos comentarios.

**Comment by zZakiraZz** [comment]
un problema que encuentro es que el rol actual es *lomata* o *lo esquiva*
